imsvdex/vdex.py

```diff
@@ -139,7 +138,9 @@ def serialize(self, file=None):
         """
         returns the vocabulary as XML
         """
-        return etree.tostring(self.tree, encoding="utf-8", standalone=True)
+        return etree.tostring(self.tree, encoding="utf-8", standalone=True).decode(
```

Comment:
Apparently this should return a native string (str in Python2 and str in Python3).
It seems to me that with this change we are returning unicode in Python2, which might be a problem.